account_payment_mode/models/account_journal.py

```diff
@@ -19,8 +19,15 @@ def _default_inbound_payment_methods(self):
         method_info = self.env[
             "account.payment.method"
         ]._get_payment_method_information()
+        manage_providers = (
+            "payment_provider_id" in self.env["account.payment.method.line"]._fields
+        )
         unique_codes = tuple(
-            code for code, info in method_info.items() if info.get("mode") == "unique"
+            code
+            for code, info in method_info.items()
+            if info.get("mode") in ("unique", "electronic")
+            and manage_providers
+            or info.get("mode") == "unique"
```

Comment:
Don't you need parenthesis in this second condition?

Reply:
Hello @pedrobaeza 
I don't think so. Can you explain your point here ?

Reply:
OK, I see, but let's make the code more readable:

```python
allowed_modes = ["unique"]
if "payment_provider_id" in self.env["account.payment.method.line"]._fields:
    allowed_modes.append("electronic")
unique_codes = tuple(
    code for code, info in method_info.items() if info.get("mode") in allowed_modes
)
```